### ISSUE TITLE
[Feature request] Always-on-top for app windows

### ISSUE THREAD

**Issue by Fan-iX** [directvt/vtm#583]
I think pinning the window to the top would be a handy feature, especially for those text editor windows.

(The placement of the trigger control may need some discretion, though.)

**Comment by o-sdn-o** [comment]
Perhaps we need to make a context menu (Mouse-Menu) for the window and for the taskbar elements and place this trigger there. This will allow users to enable “Always-On-Top” in the same way both in the window and using the taskbar. It will also be possible to bind a hotkey to this switch when keyboard support is implemented.

**Comment by Fan-iX** [comment]
Great! 

Is it possible to make a toggle for it in the terminal item bar like the `wrap` toggle for `TerminalWrapMode` ?

I tried

```
<item label=" " notes=" Always On Top " type=Option action=AlwaysOnTopApplet data=false>
    <label="*" data=true/>
</item>
```

but it didn't work.

**Comment by o-sdn-o** [comment]
Indeed, I need to add the "AlwaysOnTopApplet" action until scripting support is implemented for menu items. Thanks for the feedback! I'll add it.

**Comment by o-sdn-o** [comment]
I added `AlwaysOnTopApplet` action in v0.9.99.63 release. You can check how it works.

**Comment by o-sdn-o** [comment]
Do I also need to add the `AlwaysOnTopApplet` action for the tile manager window now? Or will that wait for scripting for window menus (I don't know how long that will take)?

**Comment by Fan-iX** [comment]
> I added `AlwaysOnTopApplet` action (for built-in terminal only) in v0.9.99.63 release. You can check how it works.

It works for me. Thanks!

> Do I also need to add the `AlwaysOnTopApplet` action for the tile manager window now? Or will that wait for scripting for window menus (I don't know how long that will take)?

I think I can wait for the implementation of window menus scripting.
